### PR TITLE
added support for new ciphers

### PR DESCRIPTION
**Pull request by deepaksibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/is_ike_policy.html.markdown [hunk 1]
```diff
@@ -40,9 +40,21 @@ resource "ibm_is_ike_policy" "example" {
 ## Argument reference
 Review the argument references that you can specify for your resource. 
 
-- `authentication_algorithm` - (Required, String) Enter the algorithm that you want to use to authenticate `IPSec` peers. Available options are `md5`, `sha1`, or `sha256`.
-- `dh_group`  - (Required, Integer) Enter the Diffie-Hellman group that you want to use for the encryption key. Available enumeration type are `2`, `5`, `14`, or `19`.
-- `encryption_algorithm` - (Required, String) Enter the algorithm that you want to use to encrypt data. Available options are: `triple_des`, `aes128`, or `aes256`.
+- `authentication_algorithm` - (Required, String) Enter the algorithm that you want to use to authenticate `IKE` peers. Available options are `md5`, `sha1`,  `sha256`, `sha512`, `sha384`.
+  
+  ~> **Note**
+  The `md5` and `sha1` algorithms have been deprecated and will be removed from November 30 2022
```
Comment (astha-jain) [inline website/docs/r/is_ike_policy.html.markdown:46]: The `md5` and `sha1` algorithms have been deprecated and support will be removed from November 2022.

File: website/docs/r/is_ike_policy.html.markdown [hunk 2]
```diff
@@ -40,9 +40,21 @@ resource "ibm_is_ike_policy" "example" {
...
+
+- `dh_group`  - (Required, Integer) Enter the Diffie-Hellman group that you want to use for the encryption key. Available enumeration type are `2`, `5`, `14`, `19`, `15`, `16` ,`17` ,`18` ,`20` ,`21` ,`22` ,`23` ,`24` ,`31`
+
+  ~> **Note**
+  The Diffie-Hellman Groups 2 and 5 have been deprecated and will be removed from November 30 2022
```
Comment (astha-jain) [inline website/docs/r/is_ike_policy.html.markdown:51]: same

File: website/docs/r/is_ike_policy.html.markdown [hunk 3]
```diff
@@ -40,9 +40,21 @@ resource "ibm_is_ike_policy" "example" {
...
+
+- `encryption_algorithm` - (Required, String) Enter the algorithm that you want to use to encrypt data. Available options are: `triple_des`, `aes128`, `aes192`, `aes256`.
+
+  ~> **Note**
+  The `triple_des` algorithm has been deprecated and will be removed from November 30 2022
```
Comment (astha-jain) [inline website/docs/r/is_ike_policy.html.markdown:56]: same

**Comment by deepaksibm** [comment]
```

$ make testacc TEST=./ibm/service/vpc TESTARGS='-run=TestAccIBMISIKEPolicy_basic'
=== RUN   TestAccIBMISIKEPolicy_basic
--- PASS: TestAccIBMISIKEPolicy_basic (63.45s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     65.178s
```

**Comment by deepaksibm** [comment]
upstream pr raised

**Pull request closed without merging**